packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -251,7 +251,8 @@ function ColorPanelDropdown( {
 						<div className="block-editor-panel-color-gradient-settings__dropdown-content">
 							{ tabs.length === 1 && (
 								<ColorPanelTab
-									{ ...tabs[ 0 ] }
+									keu={ firstTabKey }
```

Comment:
Typo? `kue`?

Reply:
I'm not seeing a warning when opening the Colors tab in global styles 🤔

Reply:
Are you sure you have SCRIPT_DEBUG true

Reply:
Also it happens only when there's just one tab in the color dropdown.

Reply:
Yes, it's true. I guess I'm not sure what you mean with dropdown, but I have one palette and multiple "elements"